src/Image.tsx

```diff
@@ -47,6 +48,8 @@ const ImageInternal: React.FC<ImageProps> = ({
   onClick,
 
   // Img
+  imgClass,
+  imgStyle,
```

Comment:
default style or className set in `img`, and need `wrapperXXX`

Reply:
ok, I have fixed it.